manifests/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -192,6 +192,8 @@ template: |
       {{ toYaml .Values.global.proxy.lifecycle | indent 4 }}
     {{- end }}
     env:
+    - name: JWT_POLICY
+      value: {{ .Values.global.jwtPolicy }}
```

Comment:
Line 391 is going to fail if your cluster doesn't have this enabled. If you want to test it just run `kind create cluster`, that will start a cluster without support. No pods will start

Reply:
Added {{- if eq .Values.global.jwtPolicy "third-party-jwt" }} before line 391.